### ISSUE TITLE
Setting the _alpha value doesn't affect the correct movieclip in The Bird and the Sea

### ISSUE THREAD

**Issue by Toad06** [ruffle-rs/ruffle#5882]
[Link to the game](http://www.ferryhalim.com/orisinal/g3/bird.htm)

When the game has finished loading, the screen fades, so the title screen becomes visible. However, unlike the Flash Player, Ruffle keeps fading the screen, making it completely white again.

---

[Sample](https://github.com/ruffle-rs/ruffle/files/7737075/fade_bug.zip)

This sample reproduces the issue.
```
Movieclip.prototype.fadescreen = function()
{
   var speed = 5;
   _root.createEmptyMovieClip("dark",100001);
   var _loc2_ = _root.dark;
   with(_loc2_)
   {
      lineStyle(1,0,100)
      beginfill(16777215,100)
      moveto(-5000,-5000)
      lineto(5000,-5000)
      lineto(5000,5000)
      lineto(-5000,5000)
      endfill()
      _loc2_.onEnterFrame = function()
      {
         _alpha = _alpha - speed;
         trace("_alpha=" + _alpha + ",_root._alpha=" + _root._alpha + ",this._alpha=" + this._alpha + ",_loc2_._alpha=" + _loc2_._alpha);
         if(_alpha <= 0)
         {
            removeMovieClip("");
            stop();
         }
      }
      
   };
};
fadescreen();
stop();
```

Flash Player traces values like:
`_alpha=-1.171875,_root._alpha=100,this._alpha=-1.171875,_loc2_._alpha=-1.171875`

Ruffle traces values like:
`_alpha=-1491.796875,_root._alpha=-1491.796875,this._alpha=100,_loc2_._alpha=100`

So as you can see:
- in Flash Player: `_alpha` == `this._alpha` == `_loc2_._alpha`.
- in Ruffle: `_alpha` == `_root._alpha`.

A workaround is to explicitly prefix all of these functions/variables with `this.` or `_loc2_.`.

**Comment by Toad06** [comment]
Also probably related to #4117.

**Comment by Toad06** [comment]
Fixed by #6082. The issue in #4117 still persists.
